tests/app/helpers.py

```diff
@@ -280,6 +280,15 @@ def set_framework_status(self, slug, status):
         Framework.query.filter_by(slug=slug).update({'status': status})
         db.session.commit()
 
+    def set_framework_variation(self, slug):
+        Framework.query.filter_by(slug=slug).update({
+            'framework_agreement_details': {
+                "frameworkAgreementVersion": "v1.0",
+                "variations": {"1": {"createdAt": "2016-08-19T15:31:00.000000Z"}}
+            }
+        })
+        db.session.commit()
+
```

Comment:
Discussed in real life: moving to use the fixture approach here would mean rewriting the whole set of tests for this endpoint and there doesn't seem to be value in doing that right now.